core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteProcedure.java

```diff
@@ -86,9 +86,14 @@ protected boolean executeAction() throws SQLException, InterruptedException {
         boolean appendOutput = convertAppendOutput(getSqlProcedure().getValue());
 
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance()
-                .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+        Connection connection;
+        try {
+            connection = ConnectionConfiguration.getInstance()
+                    .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
+                    .get();
+        } catch (Exception e) {
+            throw new RuntimeException("Unknown connection name");
+        }
```

Comment:
instead of a try-catch clause, replace .get() with orElseThrow(() -> new RuntimeException(...))

Reply:
Corrected in next commit